Failed Test: org.csstudio.archive.engine.model.MonitoredArchiveChannelUnitTest.testHandleNewValue

Jenkins test fails:

Stacktrace

java.lang.AssertionError: null
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at org.csstudio.archive.engine.model.MonitoredArchiveChannelUnitTest.testHandleNewValue(MonitoredArchiveChannelUnitTest.java:45)

--- a/applications/archive/archive-plugins/org.csstudio.archive.engine.test/src/org/csstudio/archive/engine/model/MonitoredArchiveChannelUnitTest.java
+++ b/applications/archive/archive-plugins/org.csstudio.archive.engine.test/src/org/csstudio/archive/engine/model/MonitoredArchiveChannelUnitTest.java
@@ -34,9 +34,15 @@ public class MonitoredArchiveChannelUnitTest
         final SampleBuffer samples = channel.getSampleBuffer();
         channel.start();
 
+        // Need some delay to get new time stamps, otherwise
+        // new values will be skipped since they hold the same time stamp
+        Thread.sleep(10);
         pv.write(1.0);
+        Thread.sleep(10);
         pv.write(2.0);
+        Thread.sleep(10);
         pv.write(2.05);
+        Thread.sleep(10);
         pv.write(2.5);
 
         // Allow monitors to arrive..

